script/resourceId_linux.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
these script too we can keep in forms-flow-ai repo

Reply:
Yup that makes more sense. As the role ID context is suited in forms-flow-ai